opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -606,9 +606,10 @@ def _map_dhcp_info(self, fixed_ips, mapping, mapping_dict):
                 dhcp4['lease-time'] = mapping['dhcp_lease_time']
             mapping_dict['dhcp4'] = dhcp4
             break
-        if len(v6subnets) > 0 and v6subnets.values()[0]['dns_nameservers']:
+        if len(v6subnets) > 0 and list(v6subnets.values())[0][
```

Comment:
I think this list() is indeed needed here right?

Reply:
Yes here we need because indexing is not allowed on `dict_values` type.